test_conformance/subgroups/test_barrier.cpp

```diff
@@ -147,13 +147,13 @@ int test_barrier_functions(cl_device_id device, cl_context context,
     int error;
 
     // Adjust these individually below if desired/needed
-#define G 2000
-#define L 200
+#define GWS 2000
```

Comment:
The naming is better, but it is still undocumented.
I assume by name, this is the Global Workgroup Size, in which case it should have a
// Global Work Group Size used in {...}
I'd also like a justification for this being a preprocessor macro instead of an actual constant:
constexpr uint32_t GlobalWorkSize = 2000;